File: Tsk3/Go/Subtask 2/Subtask2.go
package main

import (
	"fmt"
	"io/ioutil"
	"os"
	"path/filepath"
)

func main() {
	// Get the directory of the current script
	scriptDir, err := filepath.Abs(filepath.Dir(os.Args[0]))
	if err != nil {
		fmt.Println("Error getting the script directory:", err)
		return
	}

	// Define input and output file paths
	inputFilePath := filepath.Join(scriptDir, "input.txt")
	outputFilePath := filepath.Join(scriptDir, "output.txt")

	// Read content from the input file
	content, err := ioutil.ReadFile(inputFilePath)
	if err != nil {
		fmt.Println("Error reading input file:", err)
		return
	}

	// Write content to the output file
	err = ioutil.WriteFile(outputFilePath, content, 0644)
	if err != nil {
		fmt.Println("Error writing output file:", err)
		return
	}

	fmt.Println("File content copied successfully.")
}

File: Tsk3/Rust/Subtask 2/Subtask2.rs
use std::fs;
use std::path::PathBuf;

fn main() {
    let script_dir = PathBuf::from(std::env::current_dir().unwrap());
    let input_path = script_dir.join("input.txt");
    let output_path = script_dir.join("output.txt");
    let content = match fs::read_to_string(&input_path) {
        Ok(content) => content,
        Err(err) => {
            eprintln!("Error reading input file: {}", err);
            return;
        }
    };
    match fs::write(&output_path, content.as_bytes()) {
        Ok(_) => println!("File copied successfully!"),
        Err(err) => eprintln!("Error writing output file: {}", err),
    }
}